cylc/uiserver/resolvers.py

```diff
@@ -251,11 +255,9 @@ async def clean(
                 executor, _clean, workflow_ids, opts
             )
         except Exception as exc:
-            if isinstance(exc, ServiceFileError):  # Expected error
-                # The "workflow still running" msg is very long
-                msg = str(exc).split('\n')[0]
-            elif isinstance(exc, WorkflowFilesError):  # Expected error
+            if isinstance(exc, CylcError):  # Expected error
                 msg = str(exc)
```

Comment:
We should probs format this the same way we would for the CLI:

```suggestion
                msg = f'{exc.__class__.__name__}: {exc}'
```